app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpViewModel.kt

```diff
@@ -54,20 +60,42 @@ class NextUpViewModel(
 	private suspend fun loadItemData(id: String) = withContext(Dispatchers.IO) {
 		val item = apiClient.getItem(id) ?: return@withContext null
 
-		val thumbnail = apiClient.GetImageUrl(item, ImageOptions())
+		val thumbnail = when (userPreferences[UserPreferences.nextUpBehavior]) {
+			NextUpBehavior.EXTENDED -> apiClient.GetImageUrl(item, ImageOptions())
+			else -> null
+		}
```

Comment:
I'd prefer just sending the thumbnail but also sending the preference value and determine to show/hide in the actual fragment.

Reply:
Actually the preference should be it's own value/function in the viewmodel

Reply:
If I’m understanding correctly, wouldn’t this mean time is spent downloading the thumbnail even though it’s not shown?

Reply:
No the used function only creates a url here

Reply:
I think I'm missing something here. 😅 The thumbnail is just a url here, but it's sent to the fragment as a bitmap in `NextUpItemData`: 
```
NextUpItemData(
	...
	thumbnail?.let { safelyLoadBitmap(it) },
	logo?.let { safelyLoadBitmap(it) }
)
```
So it's already loaded the bitmap by the time the preference would get used in the fragment

Reply:
Whoops I forgot that part, you can leave it like it is then